### DISCUSSION TITLE
Can't find my pull request

### DISCUSSION THREAD

**Discussion by kevindelsh** [community/community#24266] [New to GitHub]
<p>Hi all,</p>
<p>Yesterday, I edited a file through the Github web interface. I read <a href="https://help.github.com/articles/editing-files-in-another-user-s-repository/" rel="nofollow noopener">here</a> that editing files in another user’s repository through Github’s interface would automatically forke the repository and submit a PR.</p>
<p>Right now, I have the <strong>forked repository in my account, with a branch and the updates I made but can’t find the PR in the owner’s repository</strong>.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/24c123ad841838a63cfcc827792e50c47a8d7f9b.png" data-download-href="/uploads/short-url/5f8ZgkIJ8bIVSWoLZe62U6wA95x.png?dl=1" title="Capture.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/44755032/181083772-3b469302-fcee-409b-886e-3272b5664896.png" alt="Capture.PNG" data-base62-sha1="5f8ZgkIJ8bIVSWoLZe62U6wA95x" width="690" height="68" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/24c123ad841838a63cfcc827792e50c47a8d7f9b_2_690x68.png, https://aws1.discourse-cdn.com/github/original/2X/2/24c123ad841838a63cfcc827792e50c47a8d7f9b.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/24c123ad841838a63cfcc827792e50c47a8d7f9b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/24c123ad841838a63cfcc827792e50c47a8d7f9b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.PNG</span><span class="informations">1000×99 5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>How come? Are PRs updated on a schedule or have I done anything wrong?</p>
<p>Thanks in advance.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/kevindelsh">@kevindelsh</a>,</p>
<p>Thanks for being here! Here you go <a href="https://github.com/reactjs/reactjs.org/pull/1491" rel="nofollow noopener">https://github.com/reactjs/reactjs.org/pull/1491</a>. PR’s for this repo are found at the <a href="https://github.com/reactjs/reactjs.org" rel="nofollow noopener">reactjs.org repo</a>.</p>
<p>I hope this helps!</p>
<p>Best,</p>
<p>Andrea</p>
